app/helpers/works_helper.rb

```diff
@@ -112,6 +112,35 @@ def get_inspired_by(work)
     work.approved_related_works.where(translation: false)
   end
 
+  def related_work_note(related_work, relation, download: false)
```

Comment:
Just one more thing! In the download view files and when `download` is true for this method, could you add `locale: :en` to the translation method, like we did in #4615 and #4618? We wanted to hard code the locales in downloads just to be safe.